python/semantic_kernel/connectors/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -171,6 +228,10 @@ def __init__(
         """
         if logger:
             logger.warning("The 'logger' argument is deprecated, use 'log' instead.")
+
+        if use_extensions:
```

Comment:
either note in the docstring that when use_extensions is set, that base_url is ignored, or vice versa, when base_url is set, in theory someone could create an instance with a base_url with /extensions and use_extension = False, what happens then?

Reply:
Noted in the docstring. If use_extensions is false, but base_url has /extensions, it will still call the extensions API so it's the same as use_extensions=True.